### PR TITLE
Add notes for tensors

### PR DESCRIPTION
**Pull request by terrajobst**

### CODE REVIEW

**Comment by tannergooding** [comment]
CC. @danmosemsft

File: 2019/System.Numerics.Tensors/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Adapter for System.IO.Pipelines
```
Comment (jkotas) [inline 2019/System.Numerics.Tensors/README.md]:
```suggestion
# System.Numerics.Tensors interfaces
```

Comment (terrajobst) [reply 2019/System.Numerics.Tensors/README.md]: LOL. Copy & paste at its best :-)

File: 2019/System.Numerics.Tensors/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# System.Numerics.Tensors interfaces
+
+Status: **Needs more work** | 
+[API Ref](https://github.com/dotnet/corefx/issues/35765) |
+[Video](https://www.youtube.com/watch?v=aaFcBs6cFqg)
+
+## Notes
+
+* The current proposal uses interfaces; which are very hard to version, even
+  with DIMs this might be hard due to the fact there is a hierarchy.
```
Comment (ahsonkhan) [inline 2019/System.Numerics.Tensors/README.md]: nit: for the uninitiated, call out Default Interface Methods (DIMs), especially when talking about dimensions later on :p

File: 2019/System.Numerics.Tensors/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* The original proposal used abstract classes but external libraries weren't
+  able to easily retrofit their existing types over ours; interfaces would
+  easier to inject.
+* Most libraries have their tensors baked by native memory. Given that tensors
```
Comment (eerhardt) [inline 2019/System.Numerics.Tensors/README.md]: type-o  `baked` => ` backed`

**Pull request merged**